packages/webview_flutter/example/lib/main.dart

```diff
@@ -165,37 +181,66 @@ class SampleMenu extends StatelessWidget {
               case MenuOptions.navigationDelegate:
                 _onNavigationDelegateExample(controller.data!, context);
                 break;
+              case MenuOptions.ignoreSslCertificateErrors:
+                onIgnoreSslCertificateErrors();
+                break;
             }
           },
-          itemBuilder: (BuildContext context) => <PopupMenuItem<MenuOptions>>[
+          itemBuilder: (BuildContext context) => <PopupMenuEntry<MenuOptions>>[
             PopupMenuItem<MenuOptions>(
               value: MenuOptions.showUserAgent,
-              child: const Text('Show user agent'),
+              child: ListTile(
+                leading: Icon(null),
+                title: const Text('Show user agent'),
+              ),
```

Comment:
Really hate this, but it's the advice given in [the docs for `CheckedPopupMenuItem`](https://api.flutter.dev/flutter/material/CheckedPopupMenuItem-class.html) to ensure correct alignment. It still looks pretty naff imo, but guessing it's OK because it's just an example.